cfpb/cfgov-refresh
# Add translation to the featured content module heading

To enable common use of the FCM by English and Spanish pages,
we need to translate the label without disrupting the selection
of the module icon, which depends on the English label.
Translating the label in the FCM template does the trick.

## Testing

- Load a recent db copy locally and edit the draft Spanish mortgage portal:
- http://localhost:8000/admin/pages/12693/edit/
- Add a featured content module to the page's content area and choose the category of "Featured."
- Either save as draft and view draft, or publish and view live. The FCM heading should have a star and read "Destacado" instead of "Featured"

A filled out FCM would look like this:

<img width="795" alt="featured_star_solved" src="https://user-images.githubusercontent.com/515885/57256773-86d94e80-7025-11e9-9e08-eb345d4eab97.png">
